addons/wowl/static/tests/components/dropdown_tests.ts

```diff
@@ -0,0 +1,475 @@
+import * as owl from "@odoo/owl";
+import { OwlEvent } from "@odoo/owl/dist/types/core/owl_event";
+import QUnit from "qunit";
+import { Dropdown } from "../../src/components/dropdown/dropdown";
+import { DropdownItem } from "../../src/components/dropdown/dropdown_item";
+import { OdooEnv } from "../helpers";
+import { click, makeTestEnv, mount, nextTick } from "../helpers/utility";
+
+let env: OdooEnv;
+let parent: owl.Component;
+
+QUnit.module("Dropdown", {
+  async beforeEach() {
+    env = await makeTestEnv();
+  },
+  async afterEach() {
+    parent && parent.unmount();
```

Comment:
```suggestion
    if (parent) {
        parent.unmount();
    }
```

Reply:
In fact, for this test suite, I can simply: 
```suggestion
    parent.unmount();
```